### ISSUE TITLE
Staging Review finding: Unnecessary `aria-label` for Add to calendar button

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#91569]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Appointments FE Team
**Product:** Appointments
**Feature:** Medication Review

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

The "Add to calendar" button has an `aria-label` that specifies the date of the appointment for screen reader users, eg. `aria-label="Add September 12, 2024 appointment to your calendar"`. Since the appointments details page will only ever have one appointment on it, this information is superfluous --- "Add to calendar" is fine.

Moreover, the grammar of the `aria-label` introduces a possible issue for voice command/speech recognition users. For a sighted voice user, you would expect to be able to say "Click Add to calendar" and have the button click. But voice command software searches for matches and partial matches in the accessibility tree --- and the button label is overriden by the `aria-label` in the accessibility tree. So as far as your voice command software is concerned, there is no button named "Add to calendar" to match against, only a button named "Add September 12, 2024 appointment to your calendar." But since that text is only presented to screen reader users, a sighted user would never know to say those words.

Some voice software (maybe Dragon) will probably be smart enough to recognize that as a partial match and click the button. Other voice software (probably Apple Voice Control) tends to miss on those kinds of partial matches unless what the user speaks matches the start of the `aria-label` string. In other words, `aria-label="Add to calendar your September 12, 2024 appointment"` is more likely to get a match because "Add to calendar" is how it starts.

As currently coded, I would bet most voice users aren't able to click the button via a direct command. There are workarounds for voice users, but this is a little friction we can remove pretty easily.

## Recommended action

Since this isn't doing a ton for screen reader users anyway, I think you're safe to just remove the `aria-label` entirely. If it needs to be in there for some reason, then adjust the `aria-label` text to start with "Add to calendar."

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [x] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [x] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [x] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by outerpress** [comment]
@ldelacosta #91735 completed this work, so closing this one.
